.github/workflows/frontend-command.yml

```diff
@@ -0,0 +1,124 @@
+name: "/frontend command"
+
+on:
+  repository_dispatch:
+    types: [ frontend-command ]
+
+env:
+  NODE: '14'
+
+jobs:
+  build:
+    if: ${{ github.event.client_payload.slash_command.args.unnamed.arg1 == 'build' }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - name: Checkout on chat command
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GIT_PAT }}
+          repository: ${{ github.event.client_payload.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.client_payload.pull_request.head.ref }}
+          submodules: 'recursive'
+          fetch-depth: 0
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "${{ env.NODE }}"
+
+      - name: Install node-fetch
+        run: |
+          npm install -g node-fetch@^2.6.1
+          npm link node-fetch
+
+      - name: Get build
+        id: get_build
+        env:
+          GITHUB_TOKEN: ${{ secrets.GIT_PAT }}
+          REPO: ${{ github.event.client_payload.slash_command.args.unnamed.arg2 || 'all' }}
+          SHA: ${{ github.event.client_payload.slash_command.args.unnamed.arg3 || 'master' }}
+        run: |
+          if [ "${REPO}" = "all" ]; then
+            node label_studio/frontend/get-build.js lsf
+            node label_studio/frontend/get-build.js dm
+          else
+            node label_studio/frontend/get-build.js "${REPO}" "${SHA}"
+          fi
+
+      - name: Commit and push
+        id: commit_and_push
+        shell: bash
+        run: |
+          set -xeuo pipefail
+          
+          git config --global user.name '${{ github.event.client_payload.github.actor }}'
+          git config --global user.email '${{ github.event.client_payload.github.actor }}@users.noreply.github.com'
+          
+          git add -A
+          git status -s
+          if git diff-index --quiet HEAD; then
+            echo "::set-output name=changes::no"
+            exit 0
+          else 
+            git commit -m "[frontend] Get build"
```

Comment:
Rich linkgs to commits

Reply:
Add module name in commit msg, eg Get build lsf

Reply:
done